ui/packages/components/src/Table/Cell.tsx

```diff
@@ -0,0 +1,26 @@
+import { cn } from '../utils/classNames';
+
+const cellStyles = 'text-slate-950 text-sm';
+
+export function IDCell({ children }: { children: React.ReactNode }) {
+  return <p className={cn(cellStyles, 'font-mono')}>{children}</p>;
+}
+
+export function TextCell({ children }: { children: React.ReactNode }) {
+  return <p className={cn(cellStyles, 'font-medium')}>{children}</p>;
+}
+
+export function TimeCell({ children }: { children: React.ReactNode }) {
+  // TODO: Move Time component from Cloud to shared components, to use here
+  return <span className={cn(cellStyles, 'font-medium')}>{children}</span>;
+}
+
+export function StatusCell({ status, children }: { status: string; children: React.ReactNode }) {
```

Comment:
If you get tired of manually typing `children` you can use the `React.PropsWithChildren` generic 😄 

```suggestion
export function StatusCell({ status, children }: PropsWithChildren<{ status: string }>) {
```